fiftyone/utils/ultralytics.py

```
@@ -610,6 +612,23 @@ def _forward_pass(self, imgs):
+    def predict_all(self, imgs, samples=None):
+        field_name = self._get_field()
+        if field_name is not None and samples is not None:
+            self._curr_visual_prompts = _parse_visual_prompts(
+                samples, field_name
+            )
+        else:
+            self._curr_visual_prompts = None
+
+        return self._predict_all(imgs)
+
+    def _get_field(self):
```

Ruff (0.15.1)

[warning] Missing return type annotation for private function `_get_field`

(ANN202)